static/search.less

```diff
@@ -151,3 +150,18 @@
 		position:relative;
 	}
 }
+
+/* Highlight search results */
+.highlighted {
+    position: relative;
+    display: inline-block;
+	&:after {
+		content:"";
+		position: absolute;
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
+		background-color: hsla(32, 96%, 49%, 0.61);
+	}
+}
```

Comment:
This should be taken out and only included in https://github.com/canjs/bit-docs-html-canjs/pull/261